DevCycle.SDK.Server.Common/Model/Local/DVCLocalOptions.cs

```diff
@@ -12,20 +12,26 @@ public class DVCLocalOptions : IDVCOptions
         public bool DisableAutomaticEvents { get; set; }
         public bool DisableCustomEvents { get; set; }
         public int MaxEventsInQueue { get; set; }
+        public int FlushEventQueueSize { get; set; }
         public string EventsApiUri { get; set; }
 
         public string EventsApiSlug { get; set; }
         public Dictionary<string, string> CdnCustomHeaders { get; set; }
         public Dictionary<string, string> EventsApiCustomHeaders { get; set; }
 
-        public DVCLocalOptions(int configPollingIntervalMs = 1000, int configPollingTimeoutMs = 5000,
+        public DVCLocalOptions(
+            int configPollingIntervalMs = 1000,
+            int configPollingTimeoutMs = 5000,
             string cdnUri = "https://config-cdn.devcycle.com",
             string cdnSlug = "",
             string eventsApiUri = "https://events.devcycle.com",
             string eventsApiSlug = "/v1/events/batch",
             Dictionary<string, string> cdnCustomHeaders = null,
             Dictionary<string, string> eventsApiCustomHeaders = null,
-        bool disableAutomaticEvents = false, bool disableCustomEvents = false, int maxEventsInQueue = 1000)
+            bool disableAutomaticEvents = false,
+            bool disableCustomEvents = false,
```

Comment:
These are technically handled in the AS code now for local-bucketing events.

Reply:
This constructs the config object that gets passed in to initialize the WASM code, so these do still need to be here.